src/index.js

```diff
@@ -0,0 +1,120 @@
+import './style.css';
+import setState from './modules/getstates.js';
+
+let tasks = [];
+const taskWrapper = document.querySelector('.activities');
+const newTask = document.querySelector('.newTask');
+const addNewTask = document.querySelector('.submit');
+const clearAll = document.querySelector('.clearallBtn');
+const reset = document.getElementById('reset');
+
+const clearallTasks = () => {
+  tasks = [];
+};
+
+const addToLocalStorage = () => {
+  localStorage.setItem('myTasks', JSON.stringify(tasks));
+};
+
+const getFromLocalStorage = () => {
+  if (localStorage.getItem('myTasks')) {
+    tasks = JSON.parse(localStorage.getItem('myTasks'));
+  }
+  return tasks;
+};
+
+const resetIndex = (tasks) => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    const indx = i + 1;
+    tasks[i].index = indx;
+  }
+};
+
+const rmvTask = (index) => {
+  const mylocal = getFromLocalStorage();
+  mylocal.splice(index - 1, 1);
+};
+
+const editTask = (desc, index) => {
+  tasks[index - 1].description = desc;
+  addToLocalStorage();
+};
```

Comment:
Adding function to the storage functionality is good. But the code looks better and easy to maintain if you export these functions to a separate file.

Reply:
Hello, @Dagic-zewdu thank you for this review,

- [ ] I made all the requested changes and separated the code for better readability as you had suggested.